source/documentation/permissions/evaluator.md

```diff
@@ -184,5 +184,10 @@ See the example jar for another implementation example.
         public Principal getPrincipal() {
             return principal;
         }
+	
+	@Override
+	public boolean isPrincipalAuthenticated(Object principal) {
+		return principal != null;
+	}
```

Comment:
Looks good to me, except the tabs vs. spaces here. Would you be able to update it to match the rest of the existing code example, please, @ebremer ?

Reply:
Should be fixed.  My fault for using the web interfacing and forgetting to turn off the tab indentation.